CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v2.6.1
+
+* Fixed an issue where the user location couldn't be determined on iOS 13 and older versions. ([#4113](https://github.com/mapbox/mapbox-navigation-ios/issues/4113))
```

Comment:
After the merge, this section needs to be copied over to the changelog on the main branch.